intake_vault.py

```diff
@@ -138,8 +138,7 @@ def dataset_collection_move_2_vault(ctx, toplevel_collection, dataset_id, vault_
     buffer = {}
     if status == 0:
         # stamp the vault dataset collection with additional metadata
-        date_created = datetime.now()
-        avu.set_on_coll(ctx, vault_path, "dataset_date_created", date_created.strftime('%Y-%m-%dT%H:%M:%S.%f%z'))
+        avu.set_on_coll(ctx, vault_path, "dataset_date_created", str(int(time.time())))
```

Comment:
Does this change the way dataset_date_created is stored? How is this handled for existing datasets?

Reply:
Good observation and question!
The timestamp must be stored as a number. This is the way it was originally and I introduced this fuckup myself.
I came across this error when I added the reporting part of this module to the python portal.
I erroroneously added a string formatted (Ymd etc) dataset creation timestamp.

Originally, in youth cohort ruleset, it was:

				# stamp the vault dataset collection with additional metadata
				msiGetIcatTime(*date, "unix");
				msiAddKeyVal(*kv, "dataset_date_created", *date);

As 'Youth' is not working with the new python refactored rules yet , as it was not released yet, no datasets have been added with an erroneous dataset_creation (gelukkig!)  :-)